bus/bus.go

```diff
@@ -178,6 +180,32 @@ func (b *Bus) walletSignHandler(jc jape.Context) {
 	}
 }
 
+func (b *Bus) walletSplitHandler(jc jape.Context) {
+	var wfr WalletSplitRequest
+	if jc.Decode(&wfr) != nil {
+		return
+	}
+	if wfr.Amount.Cmp(types.SiacoinPrecision) < 0 {
+		jc.Error(errors.New("'amount' has to be at least 1SC"), http.StatusBadRequest)
```

Comment:
I still don't understand why you want to enforce an arbitrary limit here

Reply:
Was not aware you were against applying these (indeed totally arbitrary) input conditions.